### DISCUSSION TITLE
Does ALDEx2 use centred-log ratio transformation in pathway_daa() function?

### DISCUSSION THREAD

**Discussion by Nazzy87** [cafferychen777/ggpicrust2#43] [Q&A] [answered]
HI,

could you be able to tell me when I use pathway_daa() for differential analysis, does it convert the picrust2 output to get centred-log ratio transformation or do I need to convert it and apply the function?

**Comment by cafferychen777** [comment, marked as answer]
Hi,

Regarding your question about using the `pathway_daa()` function for differential analysis, different DAA methods have different data preprocessing approaches. If you want to apply centred-log ratio transformation to your data, you can choose the ALDEx2 method by setting `daa_method = "ALDEx2"` in the `pathway_daa()` function.

The ALDEx2 method in `pathway_daa()` automatically performs the centred-log ratio transformation on the data, so there's no need for you to manually convert the picrust2 output before applying the function.

If you have any more questions or need further assistance, feel free to ask!

**Reply by Nazzy87** [reply]
Thank you for the reply. Actually, I have another question as well. Is there any option to draw the error bar plot and KO annotation for non-significant pathways? if the pathways that I am considering are not significantly differ between groups the code doesn't annotate and doesn't draw the error bar plot. do you have the option to annotate these non-significant pathways between groups?

**Comment by Nazzy87** [comment]
Thank you for the reply. Actually, I have another question as well. Is there any option to draw the error bar plot and KO annotation for non-significant pathways? if the pathways that I am considering are not significantly differ between groups the code doesn't annotate and doesn't draw the error bar plot. do you have the option to annotate these non-significant pathways between groups?

**Reply by cafferychen777** [reply]
Hello @Nazzy87 ,

(Maybe you can modify the pathway_errorbar function code? But it needs some R skills.

Best regards,
